documentation/docs/customjenkins.md

```diff
@@ -0,0 +1,122 @@
+# Custom Jenkins Setup
+
+Although we recommend to use the Cx server, you don't need to. You can run project "Piper" on your own Jenkins installation. However, you have to care for some settings the Cx Server gives you for free. Furthermore, the support of a none Cx Server installations is challenging. 	
+
+This section describes the necessary adjustments you might have to take.
+
+## Requirements
+
+* Java Runtime Environment 8
+* Installation of Jenkins v 2.60.3 or higher running on Linux. We tested with debian-stretch.
+* A Jenkins user with administration privileges.
+* The Jenkins instance has access to [github.com][github].
```

Comment:
```suggestion
* Your Jenkins instance has access to [github.com][GitHub].
```

Reply:
[github] is a reference only, no need for camel case.